src/MerchantCenter/MerchantCenterService.php

```diff
@@ -310,7 +311,8 @@ protected function is_mc_contact_information_setup(): bool {
 		}
 
 		if ( $contact_info instanceof AccountBusinessInformation ) {
-			$is_setup['phone_number'] = ! empty( $contact_info->getPhoneNumber() );
+			$is_setup['phone_number']              = ! empty( $contact_info->getPhoneNumber() );
+			$is_setup['phone_verification_status'] = 'VERIFIED' === $contact_info->getPhoneVerificationStatus();
```

Comment:
💅 `phone_verification_status ` seems not accurate naming for me. It sounds like we expect a status (like 'verified'. However we just want a booean.

So I would suggest 'phone_number_verified' or 'is_verified' or something like that.

Reply:
Renamed in https://github.com/woocommerce/google-listings-and-ads/pull/1732/commits/32df87f853937467b2bae809523062cb823b2abe.